examples/sample/ignition/LockModule.js

```diff
@@ -1,19 +1,19 @@
 // ./ignition/LockModule.js
 const { defineModule } = require("@ignored/hardhat-ignition");
 
-const currentTimestampInSeconds = Math.round(new Date(2023, 01, 01) / 1000);
+const currentTimestampInSeconds = Math.round(new Date(2023, 1, 1) / 1000);
 const TEN_YEAR_IN_SECS = 10 * 365 * 24 * 60 * 60;
 const TEN_YEARS_IN_FUTURE = currentTimestampInSeconds + TEN_YEAR_IN_SECS;
 
 const ONE_GWEI = hre.ethers.utils.parseUnits("1", "gwei").toString();
 
 module.exports = defineModule("LockModule", (m) => {
-  // const unlockTime = m.getOptionalParam("unlockTime", TEN_YEARS_IN_FUTURE);
-  // const lockedAmount = m.getOptionalParam("lockedAmount", ONE_GWEI);
+  const unlockTime = m.getParameter("unlockTime", TEN_YEARS_IN_FUTURE);
+  // TODO: do we intend to support runtime values for value?
+  // const lockedAmount = m.getParameter("lockedAmount", ONE_GWEI);
```

Comment:
Can I confirm we intend to expand `value` to cover runtime values?

Reply:
yes, it should support them

Reply:
I have added an issue for this enhancement here: https://github.com/NomicFoundation/ignition/issues/285